stl/src/xtime.cpp

```diff
@@ -43,28 +43,27 @@ static xtime xtime_diff(const xtime* xt,
 }
 
 
-#define EPOCH 0x19DB1DED53E8000i64
-
-#define NSEC100_PER_SEC  (NSEC_PER_SEC / 100)
-#define NSEC100_PER_MSEC (NSEC_PER_MSEC / 100)
+constexpr long long _Epoch       = 0x19DB1DED53E8000i64;

```

Comment:
Pre-existing: Ewww, an extension!
```suggestion
constexpr long long _Epoch       = 0x19DB1DED53E8000LL;
```

Reply:
Apparently, ["The i64 suffix is still supported but should be avoided because it is specific to Microsoft and is not portable."](https://docs.microsoft.com/en-us/cpp/cpp/numeric-boolean-and-pointer-literals-cpp?view=vs-2019).